### DISCUSSION TITLE
Invoice incomplete

### DISCUSSION THREAD

**Discussion by maitrepylos** [TiagoSilvaPereira/vemto-issues#454] [General]
Hello, I would like to renew my license, but in Belgium we need certain information on the invoice.
VAT number, billing address etc....
If I renew the license, is it possible to have an invoice in this format?
Thank you very much.

**Comment by TiagoSilvaPereira** [comment]
Hi @maitrepylos yes, you can add the invoice details directly on our website: https://vemto.app/invoices
